backend/api/tests/views/test_views_computetask.py

```diff
@@ -33,6 +33,8 @@ class ComputeTaskViewTests(APITestCase):
     client_class = AuthenticatedClient
 
     def setUp(self):
+        self.MaxDiff = None
```

Comment:
What does this do?

Reply:
Thanks for the heads up, that's a left over of the test debugging